tests/compatibility/check_genesis.sh

```diff
@@ -14,5 +14,5 @@ cd ${BINARY_DIR} \
     --super_admin "0x0000000000000000000000000000000000000000" \
     --nodes "127.0.0.1:4000" \
 && python3 ${SOURCE_DIR}/tests/compatibility/check_genesis.py \
-    --genesis test-chain/0/genesis.json \
+    --genesis test-chain/0/genesis.json --check_code false \
```

Comment:
is that ok? are you try to check the value of `args.check_code`?

Reply:
Done!